docs/README.md

```diff
@@ -3,6 +3,7 @@
 - [See an overview of BLS Wallet & how the components work together](./system_overview.md)
 - [Use BLS Wallet in a browser/NodeJS/Deno app](./use_bls_wallet_clients.md)
 - [Use BLS Wallet in your L2 dApp for cheaper, multi action transactions](./use_bls_wallet_dapp.md)
+- [Use BLS Wallet components and features with an Ethers-like provider](./use_bls_provider.md)
```

Comment:
```suggestion
- [Use BLS Wallet components and features with an ethers.js provider & signer](./use_bls_provider.md)
```

Reply:
Would also be good to add this entry to top level `README`

Reply:
Good shout, missed that